SUMMARY.md

```diff
@@ -22,6 +22,7 @@
 * [check-duration](miscellaneous-commands/check-duration.md)
 * [premium](miscellaneous-commands/premium.md)
 * [userconfig](miscellaneous-commands/userconfig.md)
+* [subcommand](miscellaneous-commands/subcommand.md)
```

Comment:
Wrong page name, needs to be named "Other minor commands"

File name needs to be updated to reflect